tools/Setup-Test-Environment.ps1

```diff
@@ -0,0 +1,32 @@
+param (
```

Comment:
Can you please add this under the `eng` directory instead of `tools`? We have a goal to delete the tools directory.